tests/javascript_tests/test_view_controller.js

```diff
@@ -438,5 +448,38 @@ requirejs([
         start(); // qunit
       });
     });
+
+
+    /**
+     *
+     * Test refresh metadata.
+     *
+     */
+    asyncTest('Test refreshMetadata', function() {
```

Comment:
I'm just curious, when do you anticipate using refreshMetadata?  Do you expect to pass in additional metadata during runtime?  If so,  some more detail on this would be nice to include in the docs.